source/develop/release-management/features/storage/vm-leases.html.md

```diff
@@ -105,68 +105,61 @@ Enum with these values:
 - "SHARED": lease is acquired my multiple owners (not supported yet)
 
 
-#### Lease.create(job_id, lease)
+#### Lease.create(lease)
```

Comment:
We can add it to make it look like spm verb, but would like to avoid this. The spUUID is not needed since we support only single pool.

Do you think we should keep it for consistency with other spm apis?